### DISCUSSION TITLE
Create A Database Webhook Programatically?

### DISCUSSION THREAD

**Discussion by e-lobo** [supabase/supabase#12734] [Show and tell]
I was wondering if this is possible. I do not see a way to do it though supabase.js. Maybe a rest api call then ?

**Comment by grahamorrell** [comment]
I had a need for the same thing, so did a bit of digging.  Looks like you can just create these as normal triggers :

```
CREATE TRIGGER on_YourTableName_inserted
AFTER insert on "public"."YourTableName"
FOR EACH ROW EXECUTE FUNCTION supabase_functions.http_request('https://your.api.endpoint.url', 'POST', '{"Content-type":"application/json"}', '{}', '1000')
```

Once created they will show up in the Webhooks part of the Supabase Studio and be triggered as if you'd created them through the UI
